src/modules/LaptopList/LaptopList.tsx

```diff
@@ -76,19 +76,23 @@ const LaptopList = () => {
         </LeftLayoutList>
 
         <RightLayoutList>
-          <LaptopListLayout>
-            {filteredLaptop
-              .slice((pageNum - 1) * 12, pageNum * 12)
-              .map((laptop: ILaptop) => (
-                <LaptopCard
-                  key={laptop._id}
-                  laptopId={laptop._id}
-                  image={laptop.productImg[2]}
-                  name={laptop.productName.split("(")[0]}
-                  price={laptop.price}
-                />
-              ))}
-          </LaptopListLayout>
+          {filteredLaptop.length ? (
```

Comment:
Does loading state change when the filter changed?

Reply:
I don't understand. Can you explain for me ?

Reply:
filteredLaptop doest it depend on any loading state